File: MorWalPizVideo.Models/Models/Sponsor.cs
using MongoDB.Bson.Serialization.Attributes;
using System.Runtime.Serialization;

namespace MorWalPizVideo.Server.Models
{
    [BsonIgnoreExtraElements]
    [DataContract]
    public record Sponsor([property: DataMember][property: BsonElement("title")] string Title,
        [property: DataMember][property: BsonElement("url")] string Url,
        [property: DataMember][property: BsonElement("imgSrc")] string ImgSrc) : BaseEntity
    {
    }
}

File: frontend/back-office-spa/src/models/sponsor.ts
export interface Sponsor {
  id: string;
  title: string;
  url: string;
  imgSrc: string;
  creationDateTime: string;
}

export interface CreateSponsorDTO {
  title: string;
  url: string;
  imgSrc: string;
}

export interface UpdateSponsorDTO {
  title: string;
  url: string;
  imgSrc: string;
}
